plugins-weekly.ci.jenkins.io.txt

```diff
@@ -46,7 +46,7 @@ forensics-api:2.3.0
 git:5.2.1
 git-client:4.6.0
 git-forensics:2.0.0
-github:1.37.3.1
+github:1.38.0
```

Comment:
There isn't really any changelog but https://github.com/jenkinsci/github-plugin/compare/v1.37.3.1...v1.38.0 shows only maintenance updates to work with JDK21/Guice7